docs/reference/analysis/tokenizers.asciidoc

```diff
@@ -1,6 +1,13 @@
 [[analysis-tokenizers]]
 == Tokenizer reference
 
+[NOTE]
+====
+{es}'s text analysis produces meaningful _linguistic_ tokens (like words and phrases) optimized for search relevance scoring.
+This differs from neural tokenizers, which break text into smaller subword units and numerical vectors for machine learning models.
+For example, "searching" becomes the searchable word token "search" in {es}, while a neural tokenizer might split it into ["sea", "##rch", "##ing"] for model consumption.
```

Comment:
these `##` rendered into a highlight. not sure what your intent was here but you might have to escape the chars

<img width="622" alt="image" src="https://github.com/user-attachments/assets/2df22998-71a3-42b8-bba0-189f42a089e3" />

Reply:
need to use `backticks`

Reply:
Removing the example as unnecessary detail